cpp/include/cudf/transform.hpp

```diff
@@ -63,5 +63,24 @@ std::unique_ptr<column> transform(
 std::pair<std::unique_ptr<rmm::device_buffer>, size_type>
 nans_to_nulls(column_view const& input,
               rmm::mr::device_memory_resource * mr = rmm::mr::get_default_resource());
+
+
+/**
+ * @brief Creates a null_mask from `input`, for `true` bit is set valid (`1`) for
```

Comment:
Too long for a brief.
```suggestion
 * @brief Creates a bitmask from a column of boolean elements.
 *
 * If element `i` in `input` is `true`, bit `i` in the resulting mask is set (`1`). Else,
 * if element `i` is `false` or null, bit `i` is unset (`0`). 
```